src/pages/ConfirmationPage.js

```diff
@@ -138,36 +132,30 @@ class ConfirmationPage extends React.Component {
             <Trans>We&rsquo;ve sent you a confirmation email.</Trans>
           </p>
         ) : (
-          <EmailError />
-        )} 
+          <EmailError/>
+        )}
 
         <section>
           <Confirmation
             hashFromData={this.hashFromData( email, paperFileNumber ).toString()}
             paperFileNumber={paperFileNumber}
             email={email}
-            accessibility={this.translateReason(familyCheck)}
-            privacy={this.translateReason(familyOption)}
-            location={
-              locationCity && locationAddress
-                ? locationCity + ', ' + locationAddress
-                : ''
-            }
+            accessibility={this.translateReason(accessibility)}
+            location={(locationCity && locationAddress) ? locationCity + ', ' + locationAddress : ''}
```

Comment:
Check out the back tick string syntax in es6 when you have a chance. Super clean.